examples/kubernetes/addons/etcd-operator/development/cilium-development.yaml

```diff
@@ -98,7 +98,7 @@ spec:
                   optional: true
                   key: clean-cilium-state
       containers:
-        - image: docker.io/cilium/cilium:v1.2.1
+        - image: docker.io/cilium/cilium:v1.2.2
```

Comment:
1.2.0

Reply:
Why should the development file be 1.2.0?